pkg/clients/rds/rds.go

```diff
@@ -288,31 +285,43 @@ func GenerateObservation(db rds.DBInstance) v1beta1.RDSInstanceObservation { //
 		o.DomainMemberships = make([]v1beta1.DomainMembership, len(db.DomainMemberships))
 		for i, val := range db.DomainMemberships {
 			o.DomainMemberships[i] = v1beta1.DomainMembership{
-				Domain:      aws.StringValue(val.Domain),
-				FQDN:        aws.StringValue(val.FQDN),
-				IAMRoleName: aws.StringValue(val.IAMRoleName),
-				Status:      aws.StringValue(val.Status),
+				Domain:      aws.ToString(val.Domain),
+				FQDN:        aws.ToString(val.FQDN),
+				IAMRoleName: aws.ToString(val.IAMRoleName),
+				Status:      aws.ToString(val.Status),
 			}
 		}
 	}
 	if db.Endpoint != nil {
 		o.Endpoint = v1beta1.Endpoint{
-			Address:      aws.StringValue(db.Endpoint.Address),
-			HostedZoneID: aws.StringValue(db.Endpoint.HostedZoneId),
-			Port:         int(aws.Int64Value(db.Endpoint.Port)),
+			Address:      aws.ToString(db.Endpoint.Address),
+			HostedZoneID: aws.ToString(db.Endpoint.HostedZoneId),
+			Port:         int(db.Endpoint.Port),
 		}
 	}
 	if len(db.OptionGroupMemberships) != 0 {
 		o.OptionGroupMemberships = make([]v1beta1.OptionGroupMembership, len(db.OptionGroupMemberships))
 		for i, val := range db.OptionGroupMemberships {
 			o.OptionGroupMemberships[i] = v1beta1.OptionGroupMembership{
-				OptionGroupName: aws.StringValue(val.OptionGroupName),
-				Status:          aws.StringValue(val.Status),
+				OptionGroupName: aws.ToString(val.OptionGroupName),
+				Status:          aws.ToString(val.Status),
 			}
 		}
 	}
 	if db.PendingModifiedValues != nil {
 		o.PendingModifiedValues = v1beta1.PendingModifiedValues{
+<<<<<<< HEAD
```

Comment:
I think this is a remnant from merge conflict resolution that we need to remove.